src/include/lean/lean.h

```diff
@@ -2064,11 +2066,11 @@ static inline uint8_t lean_version_get_is_release(lean_obj_arg _unit) {
 }
 
 static inline lean_obj_res lean_version_get_special_desc(lean_obj_arg _unit) {
-    return lean_mk_string(LEAN_SPECIAL_VERSION_DESC);
+    return lean_mk_ascii_string(LEAN_SPECIAL_VERSION_DESC);
 }
 
 static inline lean_obj_res lean_system_platform_target(lean_obj_arg _unit) {
-    return lean_mk_string(LEAN_PLATFORM_TARGET);
+    return lean_mk_ascii_string(LEAN_PLATFORM_TARGET);
```

Comment:
```suggestion
    return lean_mk_utf8_string(LEAN_PLATFORM_TARGET);
```
Unlikely to matter, but better safe than sorry.

Reply:
well, the "better safe than sorry" option is `lean_mk_string`. If we have any reason to believe that the data isn't necessarily in the subset it appears to be in, we should just validate it.

Reply:
Indeed, after I wrote this and was away from the computer I paused wondered if my suggestion is bogus.

It’s not entirely bogus, though: If the there is utf8 in the TARGET, then `mk_ascii_string` will produce an invalid length. So if `LEAN_SPECIAL_VERSION_DESC` etc. happens to be utf8, then `mk_utf8_string` is better than `mk_ascii_string`, but not as safe as a validating function.

I agree that it’s worth changing the `lean_mk_{utf8,utf8}_string` function names to include `unchecked to avoid future bugs.

Reply:
Agreed on better names